knimin/lib/tests/test_configuration.py

```diff
@@ -22,6 +22,14 @@ def test_init(self):
         with self.assertRaises(IOError):
             KniminConfig('does not exist')
 
+        # test that expection is raised if not all sections are specified
+        self.config = tempfile.NamedTemporaryFile()
```

Comment:
Unfortunately, it cannot be just a locale variable, because the method itself is directly using self.config and not an argument. However, since I test here the __init__ of the whole config object and trigger an exception this object should die directly after that test and should not be available anywhere else. Or am I wrong here?
I think, reading an incomplete config file is such a fundamental error, that either the system should stop right here (which it does) or be refactored!
Thus, I think we are fine with this dangerous move here, but I totally agree that we should avoid hacking an object like that in general.